cities/DK.php

```diff
@@ -0,0 +1,605 @@
+$cities['DK'] = array(
```

Comment:
You're missing the php opening and declaring cities as global.
You can check the other cities files

Reply:
Hi @Mantish – thanks!

Does it look right now? Sorry to not have got it right the first time – this is all new to me 😃

Reply:
yeah, this should work